src/startupscriptgenerator/src/node/scriptgenerator.go

```diff
@@ -44,6 +44,49 @@ const NodeWrapperPath = "/opt/node-wrapper/"
 const LocalIp = "0.0.0.0"
 const inspectParamVariableName = "ORYX_NODE_INSPECT_PARAM"
 
+// Checks if the application insights needs to be enabled for the current runtime
+func shouldApplicationInsightsBeConfigured() bool {
+	nodeAppInsightsKeyEnv := os.Getenv("APPINSIGHTS_INSTRUMENTATIONKEY")
+	nodeAppInsightsEnabledEnv := os.Getenv("APPLICATIONINSIGHTS_CODELESS_ENABLED")
+
+	if nodeAppInsightsEnabledEnv != "" && nodeAppInsightsKeyEnv != "" {
+		fmt.Printf("Environment Variables for Application Insight's Codeless Configuration exists..\n")
+		return true
+	}
+	return false
+}
+
+// Create application insights loader fiule
+func createApplicationInsightsLoaderFile(appInsightsLoaderFilePath string) {
+	appInsightsLoaderFileContent := `// --------------------------------------------------------------------------------------------
+	// Copyright (c) Microsoft Corporation. All rights reserved.
+	// Licensed under the MIT license.
+	// --------------------------------------------------------------------------------------------
+
+	// Created by Oryx
+	try 
+	{
+		var appInsights = require('applicationinsights');
+		var prefixInternalSdkVersion = function (envelope, contextObjects) {
+			try {
+				var appInsightsSDKVersion = appInsights.defaultClient.context.keys.internalSdkVersion;
+				envelope.tags[appInsightsSDKVersion] = "ad_" + envelope.tags[appInsightsSDKVersion];
+			} catch(e) {}
+
+			return true;
+		}
+		if (process.env.APPINSIGHTS_INSTRUMENTATIONKEY && process.env.APPLICATIONINSIGHTS_CODELESS_ENABLED) {
+			appInsights
+				.setup()
```

Comment:
We may want to add `.setSendLiveMetrics(true)` here since that is how it being configured for AKS. I will circle back on this.
@AlexKlimovMS @Mikhail-Pranovich

Reply:
okie,  is that confirmed then? I can add it here

Reply:
Yes confirmed you can add it